### PR TITLE
Fix pin cancer var

### PR DESCRIPTION
**Pull request by dnil**
A patch over the cancer pinned variant hgvs view. Appears some details on the genes there (and for STRs) is missing. We should return to this, but this at least just doesn't leave an empty field for pinned variants.

Screenshot 2020-02-20 at 12 48 27

Screenshot 2020-02-20 at 12 48 17

**How to test**:
1. Pin a variant of each type (cancer, cancer_sv, str, snv. sv) and make sure they render reasonably on the case page.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by
- [x] tests executed by

### CODE REVIEW

**Comment by dnil** [comment]
Mental note to fix the CHANGELOG before release 4.12.2..

**Comment by northwestwitch** [comment]
Is this on stage alerady?

**Comment by northwestwitch** [comment]
Testing on stage now. Tried to pin and mark as causative a cancer variant (the same). I've got 2 different representations at the case level:

image

Same for strs:

image



This should be quite easy to fix.

NON-cancer snvs look fine:
image

**Comment by dnil** [comment]
Right; I didn’t think about the causative part. I thought we did that earlier.. Just a moment..

**Comment by dnil** [comment]
Like so: 
Screenshot 2020-02-20 at 14 10 06

It should also really be fixed in the controller, but..

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 1]
```diff
@@ -13,19 +13,26 @@
             <div class="row align-items-between align-items-center">
               <div class="col-10">
                 <i class="fa fa-check-circle-o"></i>
-		            {% if variant.category == "snv" %}
+		            {% if variant.category in ("snv", "cancer", "str") %}
                   <a href="{{ url_for('variant.variant',
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/utils.html]: Wait a moment' won't this link be wrong if the variant is not a NON-cancer snv? then?

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/utils.html]: Hm, no?? The same variant page for all of the three, and same sv page for sv and cancer_sv?

Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/utils.html]: Yes, right, sorry! 👍

**Pull request merged**